Doc/library/os.rst

```diff
@@ -3875,28 +3960,34 @@ features:
    cancelable when if the real-time clock is changed discontinuously. Reading the descriptor
    is aborted with with the error ECANCELED.
 
+   Linux manages system clock as UTC. A daylight-savings time transition is done
+   by changing time offset only and doesn't cause discontinuous system clock change.
+
    Return a two-item tuple of (``next_expiration``, ``interval``) from
    the previous timer state, before this function executed.
 
    The following example shows how to use a timer file descriptor
    to execute a function twice a second:
 
-   .. code:: python
+   Example::
```

Comment:
@AA-Turner 
But there is no `.. code:: python` anywhere in the python code tree.
Are you suggesting new comment style here?

To avoid miscommunication, please suggest concrete code.